build/pyinstaller/terminal.spec

```diff
@@ -75,6 +75,9 @@ added_files = [
     (os.path.join(pathex, "blib2to3", "PatternGrammar.txt"), "blib2to3"),
 ]
 
+if is_win and os.path.exists(os.path.join(pathex, "scipy.libs")):
+    added_files.append((os.path.join(pathex, "scipy.libs"), "scipy.libs"))
```

Comment:
should I change this line for that? @tehcoderer

Reply:
```
if is_win:
    print("Adding scipy.libs to bundle")
    added_files.append(
        (os.path.join(f"{os.path.dirname(scipy.__file__)}.libs"), "scipy.libs/"),
    )
```